.github/workflows/ci_dev.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_TITLE: "Tag to DEV"
-          SLACK_MESSAGE: 'Starting Deployment :deployparrot:'
```

Comment:
I think you might want to configure to run eslint alone on the repo

Reply:
prettier messes up somtimes the format

Reply:
add prettier config temporarily, to help with inconsistencies. we